webapp/lib/view.dart

```diff
@@ -562,8 +552,61 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
 
   void _addTag(String tag, Element tagList) {
     // TODO: call controller.command()
-    tagList.lastChild.remove();
+    tagList.lastChild.lastChild.remove();
```

Comment:
Can we maybe keep a reference of the dropdown element so we don't have to do this, and can just do `dropdown.remove()`? There's a few reasons for this, the main one being that right here it's really not clear which element you're trying to remove, which makes reading the code and debugging harder. You'd need to have a really close understanding of the DOM layout for this to make sense.

Reply:
Okay. Made the change